### ISSUE TITLE
Punctuation level should be set to all while the Punctuation/symbol pronunciation dialog is open

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4348]
**Reported by jscholes on 2014-07-31 22:02**
While the Punctuation/symbol pronunciation dialog is open, NVDA still reads symbols according to the user's currently active punctuation level.  This means that some symbol entries are read incompletely or incorrectly.

Similar to how speech dictionary processing is disabled while editing a dictionary, punctuation level should be temporarily set suitably high for all entries to be read in the way which makes most sense.

**Comment by bhavyashah** [comment]
I continue to agree that symbol level should be automatically set to All when focus is on the Symbol Pronunciation dialog, although this may be complicated with the implementation of a unified NVDA settings dialog. However, one concern I do have is the reporting of the ';' which separates the symbol from its pronunciation as well as symbol level and so on, thus being announced thrice in the process. If the Symbol level is simply automatically changed to All, these semis would be quite obtrusive and hindering. Thoughts?

**Comment by leonardder** [comment]
[**@bhavyashah**](https://github.com/bhavyashah) commented on [12 aug. 2017 18:24 CEST](https://github.com/nvaccess/nvda/issues/4348#issuecomment-321991082 "2017-08-12T16:24:12Z - Replied by Github Reply Comments"):
> although this may be complicated with the implementation of a unified NVDA settings dialog.

Symbol pronunciation won't go into this new unified dialog. The configuration dialog will only deal with configuration manager settings (i.e. the settings that modify nvda.ini)

**Comment by bhavyashah** [comment]
@leonardder Oh, alright. Thanks for the clarification.

**Comment by Adriani90** [comment]
Closing in favor of the newer #14040.
